test/00_transform/MatMul.cu

```diff
@@ -66,8 +66,31 @@ template <typename TensorType>
 class MatMulTestFloatNonHalfTypes : public MatMulTest<TensorType> {
 };
 
+template <typename TensorType>
+class MatMulTestFloatNonComplexTypes : public MatMulTest<TensorType> {
+};
+
 TYPED_TEST_SUITE(MatMulTestFloatTypes, MatXFloatTypes);
 TYPED_TEST_SUITE(MatMulTestFloatNonHalfTypes, MatXFloatNonHalfTypes);
+TYPED_TEST_SUITE(MatMulTestFloatNonComplexTypes, MatXFloatNonComplexTypes);
+
+template <typename T>
+struct float_to_complex_t
+{
+  using t = cuda::std::complex<T>;
+};
+
+template <>
+struct float_to_complex_t<matxFp16>
+{
+  using t = matxFp16Complex;
+};
+
+template <>
+struct float_to_complex_t<matxBf16>
+{
+  using t = matxBf16Complex;
```

Comment:
Maybe call these `float_to_complex` and add a helper function that does `using float_to_complex_t = ...` to alias it and remove the need for ::t. Also can you use `type` instead of `t`?